pahfit/base.py

```diff
@@ -479,9 +498,25 @@ def save(self, obs_fit, filename, outform):
                         component.fwhm.bounds[0],
                         component.fwhm.bounds[1],
                         component.fwhm.fixed,
+                        strength,
+                        strength_unc,
                     ]
                 )
             elif comp_type == "Gaussian1D":
+
+                # Calculate line strength.
+                # TODO: Use input units to determine conversion and line strength units.
+                fwhm = 2 * component.stddev.value * np.sqrt(2 * np.log(2))
+                strength = ((np.sqrt(np.pi / np.log(2)) / np.pi) * const.c.to('micron/s').value / 2) * (component.amplitude.value * fwhm / component.mean.value**2) * 1e-26
```

Comment:
Shouldn't it just be sqrt of  pi / {ln(2)*2}? If so, one should not divide again by pi.

Reply:
I think you are right. If the others agree, I can delete and push the change.

Reply:
No idea myself.  @jdtsmith?  @ThomasSYLai?

Reply:
This should quickly fall out of comparisons with IDL PAHFIT main_feature_strength results.  Thomas and I have some new "synthetic" uncertainty-free PAH models we are working on, which should provide a nice comparison point (including Akari wavelengths).

Reply:
@jdtsmith Regarding what is currently in the IDL version, which is the aim of this PR, do you agree that it shouldn't divide again by pi? I'm only trying to move this PR forward as @karllark also stated. If there is a different approach, we should probably discuss it in an issue, or make a separate PR.